usaspending_api/recipient/management/sql/recipient_lookup/100_finalize.sql

```diff
@@ -0,0 +1,132 @@
+DO $$ BEGIN RAISE NOTICE '100 Loading recipient_lookup and cleaning up'; END $$;
+
+VACUUM ANALYZE public.temporary_restock_recipient_lookup;
+
+DO $$ BEGIN RAISE NOTICE 'Removing stale records from recipient_lookup'; END $$;
+
+BEGIN;
+WITH removed_recipients AS (
+  SELECT
+    rl.recipient_hash
+    FROM public.recipient_lookup rl
+    LEFT JOIN public.temporary_restock_recipient_lookup tem ON rl.recipient_hash = tem.recipient_hash
+    WHERE tem.recipient_hash IS NULL
+)
+DELETE FROM public.recipient_lookup rl WHERE rl.recipient_hash IN (SELECT recipient_hash FROM removed_recipients)
+RETURNING rl.recipient_hash;
```

Comment:
Is this used for anything?   When I run this script it dumps hashes out to my console for what feels like a fortnight.

Reply:
Only because the table contained 3,624,637 rows using the old hash logic. Incremental updates will contain far fewer rows.

I expect it to help with any investigations into "missing" or "changed" recipients